modules/order/commerce_order.services.yml

```diff
@@ -8,4 +8,4 @@ services:
     class: Drupal\commerce_order\Resolver\DefaultOrderTypeResolver
     arguments: ['@entity_type.manager']
     tags:
-      - { name: commerce_order.order_type_resolver, priority: 10 }
+      - { name: commerce_order.order_type_resolver, priority: -100 }
```

Comment:
Followed the pattern in https://github.com/drupalcommerce/commerce/pull/366 now that I know how priority works O:)